core/src/main/java/com/datastax/oss/driver/internal/core/metadata/DefaultTopologyMonitor.java

```diff
@@ -62,10 +61,13 @@
   // Assume topology queries never need paging
   private static final int INFINITE_PAGE_SIZE = -1;
 
+  private static final boolean EXTENDED_PEER_CHECK =
```

Comment:
I'm note sure we need this system property in driver 4. Since this component is pluggable, I think we could just enforce the extended check and if users want a more relaxed one, they can override the `isPeerValid` method.

Reply:
TBH the ticket was logged because the broadcast RPC address was null. This value is the only one that really cannot be null, otherwise we cannot connect to that peer. But the driver could survive without the checks.